perfkitbenchmarker/linux_benchmarks/hpcc_benchmark.py

```diff
@@ -165,7 +174,11 @@ def Prepare(benchmark_spec):
   PrepareHpcc(master_vm)
   CreateHpccinf(master_vm, benchmark_spec)
   CreateMachineFile(vms)
-  master_vm.RemoteCommand('cp %s/hpcc hpcc' % hpcc.HPCC_DIR)
+  if FLAGS.hpcc_binary:
```

Comment:
nitpick: Extra lines 177 - 187 to their own function since they deal with copying files around and deal with 'lower-level' details than lines from 174-176